.github/workflows/build_and_release_cuda.yml

```diff
@@ -1,6 +1,11 @@
 name: build_and_release_cuda
 
 on:
+  pull_request:
```

Comment:
Propose to filter target compute_cap in this case and run only single compute_cap

Reply:
I thought of that, wasn't sure how?